src/main/webapp/WEB-INF/views/series/info.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en" th:lang="${#locale.language == 'ru' ? 'ru' : 'en'}"
 	xmlns="http://www.w3.org/1999/xhtml"
 	xmlns:th="http://www.thymeleaf.org"
-	xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
+	xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3"
+	xmlns:togglz="https://github.com/heneke/thymeleaf-extras-togglz">>
```

Comment:
Here is the fix for an invalid syntax:
```suggestion
	xmlns:togglz="https://github.com/heneke/thymeleaf-extras-togglz">
```

Reply:
@bahoss ping me, please, when it will be ready to review.